usaspending_api/api_docs/markdown/data_dictionary.md

```diff
@@ -100,6 +100,7 @@ In this section you will find definitions for common terms used in the API. For
 | [/api/v1/awards/total/](/api/v1/awards/total/) | POST |  Aggregate (see [Using the API](/docs/using-the-api)) | Supports aggregation on award records |
 | [/api/v1/accounts/](/api/v1/accounts/) | GET, POST | <a href="#accounts">Appropriation Account Balances</a> | Returns a list of appropriation account balances |
 | [/api/v1/accounts/tas/](/api/v1/accounts/tas/) | GET, POST | <a href="#tas">Treasury Appropriation Account</a> | Returns a list of treasury appropriation accounts, by TAS |
+| [/api/v1/accounts/tas/autocomplete/](/api/v1/accounts/tas/autocomplete/) | POST | Autocomplete (see [Using the API](/docs/using-the-api))| Supports autocomplete on TAS records |
```

Comment:
We'd need to sort out a way to format the get requests, which is out of scope for this but worth a look